settings.gradle.kts

```diff
@@ -6,6 +6,9 @@ pluginManagement {
         maven(url = "http://maven.modmuss50.me") {
             name = "modmuss50"
         }
+        maven(url = "https://jitpack.io") {
```

Comment:
remove this
pluginManagement only affects gradle plugins, of which there is no changes or need for jitpack